src/Database/DBA.php

```diff
@@ -385,13 +382,13 @@ public static function rollback(): bool
 	/**
 	 * Delete a row from a table
 	 *
-	 * @param string|array $table      Table name
-	 * @param array        $conditions Field condition(s)
+	 * @param string $table      Table name
```

Comment:
Please update the documentation to reflect the type change (the possible schema prefix).